op-service/eth/ssz.go

```diff
@@ -15,7 +15,7 @@ type BlockVersion int
 
 const ( // iota is reset to 0
 	BlockV1 BlockVersion = iota
-	BlockV2              = iota
```

Comment:
With this syntax, BlockV2 is treated as int instead of `Blockversion`

Reply:
Just want to make sure - this does not change the semantics around serialization/deserialization, correct? It just makes the `BlockV2` constant a `BlockVersion`?

Reply:
Correct, and I double checked, MarshalSSZ does not serialize BlockVersion into data at all.